Chandra/test_Time.py

```diff
@@ -186,6 +186,20 @@ def test_leapsec_2015(self):
         t2 = DateTime('2015-07-01 00:00:00').secs
         self.assertAlmostEqual(t2 - t1, 1.0)
 
+    def test_leapsec_2016(self):
+        """
+        Test that there are 4 clock ticks where one usually expects 3 (PR #15).
```

Comment:
I should have updated the test docstring in b25dc78058, but the docstring is not the whole story any more.  Add something like `and that initializing within the leap sec works as expected.`  (And put the PR# commend at the end).